terraform/20-app/secret-manager.tf

```diff
@@ -226,3 +226,43 @@ resource "aws_secretsmanager_secret_version" "slack_webhook_url" {
     slack_webhook_url = ""
   })
 }
+
+################################################################################
+# UKHSA OIDC Credentials and Tenant ID
+################################################################################
+
+resource "aws_secretsmanager_secret" "ukhsa_oidc_credentials" {
```

Comment:
If we are adding secrets, then make sure they get added to the list of secrets in `scripts/_secrets.sh` that cleaned up as part of the tear down process

Reply:
Added new secrets to `_secrets.sh`